webpack.config.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
   module: {
     loaders: [{
       test: /\.js$/,
-      loaders: ['react-hot', 'babel'],
+      loaders: ['react-hot', 'babel?stage=0'],
```

Comment:
this is redundant as we have `.babelrc` file which does that, you can specify all of the babel settings there